Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/TestResultSectionView.kt

```diff
@@ -26,11 +26,10 @@ constructor(
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
 ) : ConstraintLayout(context, attrs, defStyleAttr) {
-    private val binding: ViewTestResultSectionBinding
+    private val binding = ViewTestResultSectionBinding.bind(this)
 
     init {
         inflate(context, R.layout.view_test_result_section, this)
-        binding = ViewTestResultSectionBinding.bind(this)
```

Comment:
was not this a fix? Please have a look  #2039

Reply:
Yes, that makes the app crash. The layout must first be inflated.

Reply:
good catch. had old base in my stash.
will adjust.